routing/router.go

```diff
@@ -3240,12 +3240,17 @@ func backwardPass(unifiers []*edgeUnifier, useMinAmt bool,
 	error) {
 
 	var unifiedEdges = make([]*unifiedEdge, len(unifiers))
+	var nextIncoming lnwire.MilliSatoshi
 
 	// Traverse hops backwards to accumulate fees in the running amounts.
 	for i := len(unifiers) - 1; i >= 0; i-- {
-		localChan := i == 0
 		edgeUnifier := unifiers[i]
 
+		var (
+			edge *unifiedEdge
+			fee  int64
```

Comment:
I think it will make more sense to make this an MSat value.

Reply:
Agree!

Reply:
Done!